vuu-ui/packages/vuu-popups/src/dialog/useDialog.tsx

```diff
@@ -22,6 +23,7 @@ export const useDialog = () => {
       onClose={handleClose}
       style={{ maxHeight: 500 }}
       title={dialogState.title}
+      hideCloseButton={dialogState.hideCloseButton || false}
```

Comment:
`|| false` here is redundant. hideCloseButton prop is optional in DIalogProps and Dialog has a default value (false) for it which will be used in case hideCloseButton props is left "undefined" hence here we can just pass dialogState.hideCloseButton to Dialog

Reply:
[9c6279e](https://github.com/ScottLogic/finos-vuu/pull/100/commits/9c6279ec46c085e206b24978d40182fbee811a5f)